src/Form/DeveloperSyncForm.php

```diff
@@ -46,8 +46,15 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#title' => $this->t('Sync developers'),
       '#open' => TRUE,
     ];
+
+    $form['sync']['description'] = [
+      '#markup' => '<p>' . $this->t('Developer synchronization will run through all users in this portal, adding them as developers in the Apigee Edge org, and making sure all developers on the Apigee Edge org are added to this portal. The "Run Developer Sync" button will sync the developers, displaying a progress bar. The "Background Developer Sync" button will run the developer sync process in batches each time <a href=":cron_url">cron</a> runs.', [
```

Comment:
Please use the `#html_tag` render element instead if this really needs to wrapped in `<p>` tag. Is it?

https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21Element%21HtmlTag.php/class/HtmlTag/8.5.x